terraform-plans/configs/charmed-openstack-exporter-snap_main.tfvars

```diff
@@ -12,11 +12,30 @@ templates = {
     destination = ".github/CODEOWNERS"
     vars        = {}
   }
+  check = {
+    source      = "./templates/github/snap_check.yaml.tftpl"
+    destination = ".github/workflows/check.yaml"
+    vars = {
+      runs_on = "[[ubuntu-22.04], [self-hosted, jammy, ARM64]]",
```

Comment:
I think it's okay to leave it here because I will add the support right after the func tests are merged. And we can also postpone the merge of the PR created by those changes downstream.

Reply:
See: https://github.com/canonical/charmed-openstack-exporter-snap/pull/14

Reply:
Ok no worries :+1: